### ISSUE TITLE
Options to Disable Unified Inbox

### ISSUE THREAD

**Issue by Modelmat** [Foundry376/Mailspring#38]
I really don't like the Unified Inbox, I prefer having my emails in seperate. Would it be possible to have a toggle of having the Unified Inbox disabled or at least not Ctrl+1?

**Comment by bengotow** [comment]
Hey @Modelmat — you can toggle between unified inbox and single inbox views with Ctrl-1, Ctrl-2, Ctrl-3, etc, or via this dropdown - does that do what you want? Might just need to update the knowledge base. If you switch out of Unified Inbox, it shouldn't ever switch you back automatically.

![image](https://user-images.githubusercontent.com/1037212/31258925-a923a4cc-a9f7-11e7-802f-c724f733415e.png)

**Comment by Modelmat** [comment]
Nope. I want to make it that Ctrl+1 sends me to `bengotow@gmail.com`, not `All Accounts`.

**Comment by bengotow** [comment]
Thanks for the detail—I think this is definitely possible, but we try to avoid adding features / options in the preferences until they've been requested by a bunch of folks. I'll leave this open and if folks seem interested we can pick it up!

**Comment by Rezonansowy** [comment]
Thumbs up for this! This feature is handy.

**Comment by cassvail** [comment]
Unified Inbox is useful sometimes, but other times is not. 
For example when you have many emails and many folders the list of emails and folders becomes very long and hard to manage
It would be great to add this feature!
A sample layout might be like this one (where all folders are under the same account)
<img width="202" alt="screen shot 2018-01-25 at 12 13 04" src="https://user-images.githubusercontent.com/9960733/35385996-9e4b899a-01ca-11e8-894b-bc4076eaadd4.png">
With this layout it would be possible also to have a "All Accounts" on top of all the single accounts, and inside it put the Unified Inbox, Unified Sent etc. getting **the best of both solutions**

Of course if someone prefers only the current Unified Inbox layout this could be handled in the preferences, choosing which layout to use

**Comment by Modelmat** [comment]
Also having the ability for certain accounts to be merged, not all would be great too!

**Comment by jotaf98** [comment]
Hi, similarly to @cassvail I'd like to be able to just not unify the folders. I think this makes sense when you have work and personal accounts, as many people do.

Switching between accounts with Cmd+2/3/etc is a step in the right direction, but introduces unnecessary friction. What about just showing all accounts separately, but at the same time? (Using the shortcuts hides all but one account.)

**Comment by iandunn** [comment]
I'd also like to see all folders at once, but not merged. I have 4 accounts setup, and each has between 3 and 10 custom folders. With the unified view, some of the folders are merged, while other's aren't (because they don't have the same name/purpose). I dislike the unified mailboxes, because each account has separate purposes, and merging them creates a lot of disorganization and clutter. If I wanted all my email in one place, I'd just use a single account instead.

Manually switching between accounts becomes annoying, and it hides the status of all the other accounts. I can't tell if any of the folders have new messages, unless I manually cycle though each account. I disable all notifications, because they're distracting and unnecessary (not just Mailspring's but all apps notifications). Even if notifications were enabled, though, they could have appeared and automatically disappeared while I was away from my desk, etc.

I just want to see everything all that once, the way that Thunderbird and other mail clients do it. That might make the folder bar scroll, but that could probably be fixed by letting me hide IMAP folders I don't use, and features I don't use, like "Reminders", "Snoozed", "Activity", etc.

**Comment by rlefaivre** [comment]
Hi,

I just got to Mailspring and I love it ! This is an awesome job !

Like many above, I'd like to switch off the Unified mode.
For me, this would help getting each mailbox fully separated, but still visible and accessible in a single click !

Thanks !

**Comment by why-not** [comment]
One more vote for switching off Unified inbox. I go to a lot of trouble to have multiple inboxes. I don't want to mix all of that. Cmd + 1 should be what ever I want it to be, but at minimum unified inbox should be disable-able.

**Comment by calebcook** [comment]
I agree that there should be a preference to disable Unified Inbox, and possibly an option to exclude certain accounts from a Unified Inbox if enabled.

**Comment by jcapo** [comment]
+1 for being able to disable the Unified Inbox.

**Comment by lakawak** [comment]
I have to be honest, I find it hard to believe that ANYONE wants all their e-mails merged into 1 inbox. That seems like a huge step BACKWARDS...like to 1990s e-mail.  mean, there is a reason why EVERY OTHER mail client allows you to see all your accounts on the left, even when they also have a merged inbox that you can use if you want. And sometimes, when every other one is doing something, it is because it is by far the best. I just tried Mailspring and this warranted an instant uninstall. All for this. If I wanted all my e-mails merged, then I wouldn't HAVE multiple accounts in the first place. Some accounts are more important than others and require my attention quicker. I don't want to have that inbox bogged down with e-mails from the others. No do I want to have to switch back and forth between the ones I want to see.

I seriously can't imagine how the developers thought it was a good idea. And even more strange, that they hype it as a feature! And as a PRODUCTIVITY feature!

**Comment by mikeytwice** [comment]
>  If I wanted all my e-mails merged, then I wouldn't HAVE multiple accounts in the first place. Some accounts are more important than others and require my attention quicker. 

+1. I'd like to see something more along the lines of how many other email clients (Airmail, Apple Mail) work. Even just being able to move All Accounts to the bottom of the left-side pane would help, though ideally I'd like some visual distinction. (Airmail does this well, but I hate the composer and search functionality.)

**Comment by edwardzhxi** [comment]
> I'd also like to see all folders at once, but not merged. I have 4 accounts setup, and each has between 3 and 10 custom folders. With the unified view, some of the folders are merged, while other's aren't (because they don't have the same name/purpose). I dislike the unified mailboxes, because each account has separate purposes, and merging them creates a lot of disorganization and clutter. If I wanted all my email in one place, I'd just use a single account instead.
> 
> Manually switching between accounts becomes annoying, and it hides the status of all the other accounts. I can't tell if any of the folders have new messages, unless I manually cycle though each account. I disable all notifications, because they're distracting and unnecessary (not just Mailspring's but all apps notifications). Even if notifications were enabled, though, they could have appeared and automatically disappeared while I was away from my desk, etc.
> 
> I just want to see everything all that once, the way that Thunderbird and other mail clients do it. That might make the folder bar scroll, but that could probably be fixed by letting me hide IMAP folders I don't use, and features I don't use, like "Reminders", "Snoozed", "Activity", etc.
> 
> I like some things about Mailspring, but this is a dealbreaker for me.

totally agree!!!
we should be able to disable it!

**Comment by petsasj** [comment]
Just jumping in to say that i'd really like this feature as well

I am not a fan of unified inbox (in general, not Mailspring's implementation), and like to keep everything separated

**Comment by rsmsha** [comment]
I can't believe that its been 2 years and a future as simple as this one with quite a few requests hasn't been implemented.

**Comment by tpluscode** [comment]
Yes please, make this possible!

**Comment by sikandarchishty** [comment]
Hi,
Please remove unified inbox. It creates a lot of fuss seeing work and private inboxes at the same time.
Thanks.

**Comment by romgen** [comment]
For everyone who doesn't like the unnified inbox: Please use a different mail client. Mailspring is one of the last clients to support a unified inbox. If you don't like it, you can switch to e.g. Outlook.

**Comment by mikeytwice** [comment]
Except for all the other clients that support it (but don't force you to use it), like Spark, Airmail, etc.

**Comment by romgen** [comment]
As I said: Feel free ot use the other clients...

**Comment by CodeMouse92** [comment]
We are in the process of migrating issues to Discourse, which can better facilitate discussion and discovery, and so GitHub Issues can focus on issues that are confirmed and slated for resolution in the near term. [Learn more about the changes here.](https://community.getmailspring.com/t/mailspring-community-transition/38)

As part of this, we've migrated this issue to Discourse:

https://community.getmailspring.com/t/disable-selective-unified-inbox/136

Please consider joining that community and continuing the discussion there! **Votes on the feature suggestion on Discourse will increase the likelihood we implement this.**

OP: if you join and reply to the issue, the moderators can make an effort to reassign the post to you, so you get the credit for it.

We're closing and locking the issue here as part of this migration. Rest assured, this doesn't mean the issue is being discarded or ignored.

We hope to see you on Discourse soon!

-The Mailspring Team
